src/classes/presentation_controller.vala

```diff
@@ -1214,8 +1214,8 @@ namespace pdfpc {
          * controllable's main view.  Also, return the XID for the view's window,
          * useful for overlays.
          */
-        public uint* overlay_pos(int n, Poppler.Rectangle area, out Gdk.Rectangle rect) {
-            Controllable c = this.controllables.nth_data(n);
+        public uint* overlay_pos(int n, Poppler.Rectangle area, out Gdk.Rectangle rect, ref int gdk_scale) {
```

Comment:
The current signature with `rect` declared `out` and `gdk_scale` declared `ref` seems somewhat arbitrary to me. Is there a particular reason for choosing `ref` with outside initialization to `1` over declaring `gdk_scale` as another `out` variable and moving the initialization into the function?